packages/engine-ui/src/components/forms/inputs/FleetInput.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react'
+import * as Elements from '../../../shared-elements'
+import * as stores from '../../../utils/state/stores'
+import styled from 'styled-components'
+import arrowIcon from '../../../assets/input-arrowdown.svg'
+
+const ShowDropdownIcon = styled.img`
+  width: 10px;
+  height: 10px;
+`
+const ShowDropdownButton = styled.button`
+  position: absolute;
+  left: 90%;
+  height: 90%;
+  top: 2px;
+  border: none;
+  font-weight: 600;
+  color: ${({ color }) => color || 'inherit'};
+  font-size: 0.875rem;
+  border: none;
+  cursor: pointer;
+  background-color: #f1f3f5;
+  :focus {
+    outline: none;
+  }
+`
+
+const DropdownButtonWithAddButton = styled(Elements.Form.DropdownButton)`
+  display: flex;
+  justify-content: space-between;
+  button {
+    font-weight: 600;
+    color: ${({ color }) => color || 'inherit'};
+    font-size: 0.875rem;
+    border: none;
+    cursor: pointer;
+    background-color: #f1f3f5;
+    :focus {
+      outline: none;
+    }
+  }
+`
+
+const Dropdown: React.FC<{
+  items: string[]
+  searchValue?: string
+  handleDropdownSelect: (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    value: string
+  ) => void
+}> = ({ items, handleDropdownSelect, searchValue }) => {
+  console.log(searchValue)
```

Comment:
Maybe remove this console.log?

Reply:
I saw that, removed in the latest commit :)